tests/CMakeLists.txt

```diff
@@ -1,41 +1,53 @@
-cmake_minimum_required( VERSION 3.5 )
+cmake_minimum_required(VERSION 3.5)
 
-set(EOSIO_VERSION_MIN "2.0")
-set(EOSIO_VERSION_SOFT_MAX "3.0")
-#set(EOSIO_VERSION_HARD_MAX "")
+set(EOSIO_VERSION_MIN "3.1")
+set(EOSIO_VERSION_SOFT_MAX "3.1")
+# set(EOSIO_VERSION_HARD_MAX "")
 
 find_package(eosio)
 
-### Check the version of eosio
+# Check the version of eosio
 set(VERSION_MATCH_ERROR_MSG "")
-EOSIO_CHECK_VERSION(VERSION_OUTPUT "${EOSIO_VERSION}"
-                                   "${EOSIO_VERSION_MIN}"
-                                   "${EOSIO_VERSION_SOFT_MAX}"
-                                   "${EOSIO_VERSION_HARD_MAX}"
-                                   VERSION_MATCH_ERROR_MSG)
+eosio_check_version(VERSION_OUTPUT "${EOSIO_VERSION}" "${EOSIO_VERSION_MIN}" "${EOSIO_VERSION_SOFT_MAX}"
+                    "${EOSIO_VERSION_HARD_MAX}" VERSION_MATCH_ERROR_MSG)
 if(VERSION_OUTPUT STREQUAL "MATCH")
-   message(STATUS "Using eosio version ${EOSIO_VERSION}")
+  message(STATUS "Using eosio version ${EOSIO_VERSION}")
```

Comment:
Do we want to change this to text to Mandel version? I'm fine with EOSIO_VERSION still being there for now.

Reply:
Again, I'll wait until the Mandel rename PR for all these to be tackled consistently.